src/angularComponent/ngcOmniboxComponent.js

```diff
@@ -0,0 +1,6 @@
+import NgcOmniboxController from './ngcOmniboxController.js';
```

Comment:
How do ui-select or ui-bootstrap do it? I imagine they enlist the help of their build tools to add their templates to `$templateCache` the way we used to do it in src/angularapps?